### DISCUSSION TITLE
How display nginx error page with flask ?

### DISCUSSION THREAD

**Discussion by BobSquarePants** [pallets/flask#4730] [Q&A]
Hi everyone,

I've just installed and setup nginx in front of my Flask/Gunicorn

While test it I see that some error page are handled by nginx (that make sense)
![beurk](https://user-images.githubusercontent.com/23012674/182561802-b631b147-2ba2-4178-8b3e-3061728194af.png)

I've see that we can define custom page in the nginx config.
But all the example that I've seen are static.

is there a way the set one page/route in flask that will grab the error code ?
Then when can use within the page with jinja for example.

Thanks.

**Comment by zhenzi0322** [comment]
Yes, you can.
```python
...
@app.errorhandler(404)
def handle_404_error(error):
    print(error)
```

**Reply by BobSquarePants** [reply]
I have already something similar but that catch any ! error --

```python
@app.errorhandler(HTTPException) 
def handle_exception(e):
	return "your error" + e
```
and even with this one, the Nginx appear anyway.
